### PR TITLE
Implement Geodesy definition

### PR DESCRIPTION
**Pull request by DenisCarriere**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by sheetalkamat** [comment]
@DenisCarriere "strictNullChecks" need to be true since this is new package

**Comment by sheetalkamat** [comment]
Also please fix the test

File: geodesy/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+declare class Dms {
+    static parseDMS(dmsStr: string): number;
+    static toDMS(deg: number, format?: format, dp?: 0 | 2 | 4): string;
+    static toLat(deg: number, format?: format, dp?: 0 | 2 | 4): string;
+    static toLon(deg: number, format?: format, dp?: 0 | 2 | 4): string;
+    static toBrng(deg: number, format?: format, dp?: 0 | 2 | 4): string;
+    static compassPoint(bearing: number, precision?: 1 | 2 | 3): string;
+}
+export { Dms }
```
Comment (andy-ms) [inline geodesy/index.d.ts]: Just export the class with `export class Dms { ... }`.

Comment (DenisCarriere) [inline geodesy/index.d.ts]: Cool trick! I didn't know you could export both of them, it was causing an error when I only did one at a time.

**Comment by DenisCarriere** [comment]
@andy-ms @sheetalkamat This next commit should fix the concerns you both mentioned.

Thanks for the quick feedback 👍

**Pull request merged**